src/main/resources/session-template-data/session-data.csv

```diff
@@ -0,0 +1,15 @@
+PRISON,ROOM,TYPE,OPEN,CLOSED,START_TIME,END_TIME,START_DATE,END_DATE,DAY_OF_WEEK,LOCATION_KEYS
+HEI,Visits Main Room,SOCIAL,30,2,13:45:00,14:45:00,2022-05-30,,MONDAY,
+HEI,Visits Main Room,SOCIAL,30,2,13:45:00,14:45:00, 2022-06-01,,WEDNESDAY,
+HEI,Visits Main Room,SOCIAL,30,2,09:00:00,14:00:00, 2022-06-03,,FRIDAY,
```

Comment:
This line doesn't look right - maybe a mistake in the CSV file

Reply:
@rj-adams would be good if you could look at the generated SQL file 

`/src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql`

Reply:
spaces not a problem but on line 4 the start of the session is 0900 but the end of session is 1400 - a 5hr hour meet?!?

HEI,Visits Main Room,SOCIAL,30,2,**_09:00:00,14:00:00_**, 2022-06-03,,FRIDAY,

Reply:
Oh I got you

Reply:
updated,... will spend a faire bit of time fixing this tomorrow!